x-pack/plugins/fleet/common/types/rest_spec/fleet_setup.ts

```diff
@@ -5,8 +5,9 @@
  * 2.0.
  */
 
-export interface CreateFleetSetupResponse {
```

Comment:
`CreateFleetSetupResponse` was only used in a test script for another plugin and was essentially the same as `PostIngestSetupResponse` in `./ingest_setup`